dts/bindings/pwm/nxp,lpc-pwm.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2017, NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: NXP LPC PWM
+
+compatible: "nxp,lpc-pwm"
```

Comment:
why "nxp,lpc-pwm".  seems like we should have sctimer in the name.

Reply:
I have renamed the driver to `pwm_mcux_sctimer.c` and changed this to `nxp_sctimer_pwm`